stable_plugins/classical_ml/data_preparation/aggregators.py

```diff
@@ -276,7 +297,55 @@ def calculation_task(self, db_id: int) -> str:
         # removes .json from file name to get the name of the attribute
         attr_name = file_name[:-5]
 
-        attribute_distances[attr_name] = json.load(file)
+        loaded_distances = json.load(file)
+
+        if missing_data_handling == MissingDataHandling.ignore:
+            # removes elements with None distance
+            loaded_distances = [
+                dist for dist in loaded_distances if dist["distance"] is not None
+            ]  # FIXME: handle all distances being None
```

Comment:
FIXME can be removed now...

Reply:
I added these fixmes, because it would still be a problem if all distances of an attribute are None or if the distances for all attributes of an entity pair are None.

Reply:
But those cases cannot be handled in this plugin. (Or at least I don't see how they could be)